### PR TITLE
[SYCLomatic] Update the DPCT1099 message

### PR DESCRIPTION
**Pull request by zhiweij1**
Signed-off-by: Jiang, Zhiwei <zhiwei.jiang@intel.com>

### CODE REVIEW

**Review by tomflinda — APPROVED** [review]
LGTM

File: clang/test/dpct/cufft-deduce.cu [hunk 1]
```diff
@@ -58,7 +58,7 @@ int main() {
   //CHECK-NEXT:DPCT1100:{{[0-9]+}}: Currently the DFT external workspace feature in the Intel(R) oneAPI Math Kernel Library (oneMKL) is only supported on GPU devices. Use the internal workspace if your code should run on non-GPU devices.
   //CHECK-NEXT:*/
   //CHECK-NEXT:/*
-  //CHECK-NEXT:DPCT1099:{{[0-9]+}}: Verify if the default value of the direction and placement used in the function "commit" is correct.
+  //CHECK-NEXT:DPCT1099:{{[0-9]+}}: Verify if the default value of the direction and placement used in the function "commit" is aligning with the related "compute" function call.
```
Comment (zhimingwang36) [inline clang/test/dpct/cufft-deduce.cu]: Verify if the default value of the direction and placement used in the function "commit" aligns with the related "compute" function call

Comment (zhiweij1) [reply clang/test/dpct/cufft-deduce.cu]: done

**Review by zhimingwang36 — APPROVED** [review]
LGTM

**Pull request merged**